<?xml version="1.0"?>
<robot name="mrobot" xmlns:xacro="http://www.ros.org/wiki/xacro">

	<xacro:include filename="$(find agv_robot)/urdf/robot_body.urdf.xacro" />
	<xacro:include filename="$(find agv_robot)/urdf/rplidar.xacro" />	

	<xacro:property name="rplidar_offset_x" value="0" />
	<xacro:property name="rplidar_offset_y" value="0.29" />
	<xacro:property name="rplidar_offset_z" value="0.33" />

	<!-- MRobot机器人平台-->
	<mrobot_body/>

	<!-- rplidar -->
	<joint name="rplidar_joint" type="fixed">
		<origin xyz="${rplidar_offset_x} ${rplidar_offset_y} ${rplidar_offset_z}" rpy="0 0 0" />
		<parent link="base_link"/>
		<child link="base_laser"/>
	</joint>
	<xacro:rplidar prefix="base_laser"/>

</robot>
